templates/home.html

```diff
@@ -11,17 +11,82 @@
         <div id="home-page" class="home-page">
             <style>
                 #home-page {
+                    background: linear-gradient(
+                        rgba(0, 0, 0, 0.5),
+                        rgba(0, 0, 0, 0.5)
+                        ), url("{% static "img/perdiem-background.jpg" %}");
+                    background-size: cover;
+                    background-repeat: no-repeat;
                     background-color: black;
                 }
             </style>
-            <div class="home-description">
-                <div class="footer-position auth">
-                    <h4 class="white-color header-padding">Welcome, {% firstof user.get_full_name user.get_username %}</h4>
-                    <div class="discover">
-                        <div class="center extra-padding">
-                            <a class="button sign-up uppercase" href="{% url 'profile' %}">Go to profile</a>
-                        </div>
+            <div class="header-position">
+                <h2 class="white-color header-text">Welcome, {% firstof user.get_full_name user.get_username %}</h2>
+                <h4 class="white-color header-padding">See how your artists are doing</h4>
+                <div class="center extra-padding">
+                    <a class="button sign-up uppercase" href="{% url 'profile' %}"><h4 class="white-color">View Profile</h4></a>
+                </div>
+            </div>
+            <div class="footer-position">
+                <a href={% url 'home' %}#about>
+                    <h5>Learn More</h5>
+                    <i class="fa fa-chevron-down fa-1x color-grey" aria-hidden="true"></i>
+                </a>
+            </div>
+        </div>
+        <div id="about" class="about">
+            <div id="column-wrapper">
+                <div id="leftcolumn">
+                    <img class="img-phone" src="{% static "img/perdiem-invest.png" %}">
+                </div>
+                <div id="rightcolumn">
+                    <h3 class="title">Buy and sell shares in music</h3>
+                    <p>
+                        <strong>We make it easy for anyone to invest in music.</strong> Never worry about contracts or legal fees. Artists can raise money by selling shares in their music projects. This gives people an incentive to fund the artists music as they can earn a return on their investment based on amount of sales.
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div id="how-page" class="about">
+                <div id="column-wrapper">
```

Comment:
Indenting is off here. Looks like this whole section should be unindented 4 spaces.